packages/design-preview/src/components/sidebar.tsx

```diff
@@ -33,7 +33,8 @@ const Sidebar: React.FC< SidebarProps > = ( {
 
 			{ variations.length > 0 && (
 				<div className="design-preview__sidebar-variations">
-					<h2> { translate( 'Style variations' ) }</h2>
+					<h2>{ translate( 'Style variations' ) }</h2>
+					<h3>{ translate( 'Choose a variation to change the look of the site.' ) }</h3>
```

Comment:
Super nit: Semantically I don't think this is a heading; I think this is more of a `<p>`. What do you think?

(At the very least, it is weird that currently this is a `<h3>` but the font size is smaller than the `<p>` from the theme's description right above it)

Reply:
Sounds good to me, updated 🙂